drivers/include/sx126x.h

```diff
@@ -42,19 +56,22 @@ typedef struct {
     gpio_t busy_pin;                    /**< Busy pin */
     gpio_t dio1_pin;                    /**< Dio1 pin */
     sx126x_reg_mod_t regulator;         /**< Power regulator mode */
+    void(*set_rf_mode)(sx126x_t *dev, sx126x_rf_mode_t rf_mode); /**< Interface
+    to set RF switch parameters */
 } sx126x_params_t;
 
 /**
  * @brief   Device descriptor for the driver
  */
-typedef struct {
+struct sx126x {
     netdev_t netdev;                        /**< Netdev parent struct */
     sx126x_params_t *params;                /**< Initialization parameters */
     sx126x_pkt_params_lora_t pkt_params;    /**< Lora packet parameters */
     sx126x_mod_params_lora_t mod_params;    /**< Lora modulation parameters */
     uint32_t channel;                       /**< Current channel frequency (in Hz) */
     uint32_t rx_timeout;                    /**< RX timeout in ms */
-} sx126x_t;
+    bool radio_sleep;                       /**< Sub-GHz radio sleep status */
```

Comment:
```suggestion
    bool radio_sleep;                       /**< radio sleep status */
```

This will be also required by NETOPT_STATE_SLEEP